docs/angular/platform.md

```diff
@@ -54,6 +54,42 @@ Below is a table listing all the possible platform values along with correspondi
 | pwa           | a PWA app                                |
 | tablet        | a tablet device                          |
 
+#### Customizing Platform Detection Methods
+
+The method used to detect a specific platform can also be overridden by providing an alternative method in the global Ionic [config](/docs/react/config). Each method takes `window` as a parameter and returns a boolean. 
+
+```tsx
+import { IonicModule } from '@ionic/angular';
+
+@NgModule({
+  ...
+  imports: [
+    BrowserModule,
+    IonicModule.forRoot({
+      platform: {
+        /** The default `desktop` method returns false for devices with a touchscreen. 
+        * This is not always wanted, so this method tests the User Agent instead.
+        **/
+        'desktop': (win) => {
+          const isMobile = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(win.navigator.userAgent);
+          return !isMobile;
+        } as PlatformConfig
```

Comment:
Why do we need to typecast this as `PlatformConfig`? `forRoot` accepts the type `IonicConfig` and `PlatformConfig` is used in that type.